python/lancedb/table.py

```diff
@@ -292,7 +292,9 @@ def delete(self, where: str):
         --------
         >>> import lancedb
         >>> import pandas as pd
```

Comment:
I think we don't need the pandas import anymore.

Reply:
Removed.